deployment/stacks/ui.ts

```diff
@@ -61,6 +61,26 @@ export function createUiComponents(props: CreateUiComponentsProps) {
       removalPolicy: RemovalPolicy.RETAIN,
       enforceSSL: true,
       versioned: true,
+      lifecycleRules: [
+        {
+          expiration: Duration.days(1095),
+          noncurrentVersionExpiration: Duration.days(1095),
+        },
+      ],
+    });
+
+    logBucket.addLifecycleRule({
+      id: "ExpiredLogs",
+      enabled: true,
+      expiration: Duration.days(1095),
+      noncurrentVersionExpiration: Duration.days(1095),
+      abortIncompleteMultipartUploadAfter: Duration.days(1095),
+    });
+
+    logBucket.addLifecycleRule({
+      id: "CleanUpDeleteMarkers",
+      enabled: true,
+      noncurrentVersionExpiration: Duration.days(1095),
     });
```

Comment:
```suggestion
      lifecycleRules: [
        {
          expiration: Duration.days(1095),
          noncurrentVersionExpiration: Duration.days(1095),
        },
      ],
    });
```
I think it's an either/or vs doing it in both ways.